lib/app/error_reporting.dart

```diff
@@ -126,7 +125,7 @@ Future<bool> reportEvent(Event event) async {
 
 Future<Contexts> _getContexts() async {
   try {
-    final packageInfo = await PackageInfo.fromPlatform();
+    final packageInfo = services.get<PackageInfo>();
```

Comment:
Shouldn't we also add a `services.packageInfo` getter just like for all the other services?